src/components/ComputeModal.js

```diff
@@ -664,6 +690,9 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
       const { computeZone, computeRegion } = getRegionInfo(location || defaultLocation, locationType)
       const runtimeConfig = currentRuntimeDetails?.runtimeConfig
       const gpuConfig = runtimeConfig?.gpuConfig
+      const autopauseConfig = !! currentRuntimeDetails ?
```

Comment:
We typically don't put space between unary operators (like `!`) and operands. Note that `!!` is not a separate operator but two `!` operators, one after another.
```suggestion
      const autopauseConfig = !!currentRuntimeDetails ?
```